ArduPlane/quadplane.cpp

```diff
@@ -2577,9 +2577,6 @@ void QuadPlane::vtol_position_controller(void)
                 const float gnd_speed = plane.ahrs.groundspeed();
                 Vector2f target_speed_xy = landing_velocity + diff_wp_norm * gnd_speed - wind;
                 target_yaw_deg = degrees(target_speed_xy.angle());
-
-                // now limit speed to scaled wp speed, so we don't fly rapidly backwards
-                target_speed_xy.limit_length(scaled_wp_speed*100);
```

Comment:
I wonder if it should have been this:
```target_speed_xy_cms.limit_length(scaled_wp_speed*100);```

`scaled_wp_speed*100` is certainty in cm/s. 

However using `scaled_wp_speed` is nonsense because it is based on pointing nose towards the destination, which this new code specifically does not do rather pointing nose into the apparent wind.

So on the whole I think removing this dead line is correct.